### PR TITLE
fix scalar dispute cards outcome group view

### LINKED ISSUE #4229: Market Cards: scalar market cards in-reporting should show 2 outcomes like binary
an outcome shouldn't take up the whole line it should have two outcomes on one line with `show more`

image

like this market card

image

with the `show more`

### PR DESCRIPTION
**Pull request by phoebemirman**
closes https://github.com/AugurProject/augur/issues/4229

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://featurepeek.com). 🙌

 This link will always point to this PR's latest changes: https://peek.run/l74h37n

File: packages/augur-ui/src/modules/market-cards/market-card.tsx [hunk 1]
```diff
@@ -143,31 +144,31 @@ export default class MarketCard extends React.Component<
       <>
         {address && address.toUpperCase() === author.toUpperCase() && (
           <HoverIcon
-            label='marketCreator'
+            label="marketCreator"
```
Comment (bthaile) [inline packages/augur-ui/src/modules/market-cards/market-card.tsx]: any reason why these are now double quotes?

File: packages/augur-ui/src/modules/market-cards/market-card.tsx [hunk 2]
```diff
@@ -299,7 +303,7 @@ export default class MarketCard extends React.Component<
                 !expandedView && (
                   <button onClick={this.expand}>
                     <ChevronFlip
-                      stroke='#fff'
+                      stroke="#fff"
```
Comment (bthaile) [inline packages/augur-ui/src/modules/market-cards/market-card.tsx]: formatter got confused, I'm guessing

**Review by bthaile — CHANGES_REQUESTED** [review]
use single quotes, your formatter might have gotten confused

**Review by bthaile — APPROVED** [review]
lgtm

**Pull request merged**